src/index.js

```diff
@@ -0,0 +1,146 @@
+const state = {
+    city: 'Honolulu',
+    lat: 21.3099,
+    long: 157.8581,
+    temp: 72,
+};
+
+const convertKtoF = (temp) => {
+    return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const findLatitudeAndLongitude = () => {
+    let location = state.city;
+    axios.get('http://localhost:5000/location',
+    {
+        params: {
+            q: location,
+        },
+    })
+    .then( (response) => {
+        console.log(response);
+        state.lat = response.data[0].lat;
+        state.long = response.data[0].lon;
+        console.log(`response: ${state.city}`);
+        console.log('success in findLatitudeAndLongitude!', state.lat, state.long);
+        
+        getWeather();
+    })
+    .catch( (error) => {
+    console.log(`error: ${state.city}`);
+    console.log('error in findLatitudeAndLongitude!');
+    console.log(error);
+    });
+};
+
+const getWeather = () => {
+    axios
+      .get('http://localhost:5000/weather', {
+        params: {
+          lat: state.lat,
+          lon: state.long,
+        },
+      })
+      .then((response) => {
+        const weather = response.data;
+        state.temp = Math.round(convertKtoF(weather.main.temp));
+        formatTempAndGarden();
+      })
+      .catch((error) => {
+        console.log('Error getting the weather:', error);
+      });
+};
+
+// --------------------------------------------------------------------------------------
+
+const updateSky = () => {
+    let sky = document.getElementById('sky');
+    let skyOption = document.getElementById('sky-select').value;
+    if (skyOption === 'Sunny') {
+        sky.textContent = "☁️☁️✨🌚☁️🌝☁️✨☁️";
+    } else if (skyOption === 'Cloudy') {
+        sky.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+    } else if (skyOption === 'Rainy') {
+        sky.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+    } else if (skyOption === 'Snowy') {
+        sky.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+    }
```

Comment:
You could use a value that has skyOptions as keys and the sky emojis as values so that you could iterate over the object to get the values you need to display a new sky. Iterating over an object would decrease code complexity by removing if/else if branching that happens in this method.